packages/connection-router-api/src/msg.rs

```diff
@@ -48,4 +48,10 @@ pub enum ExecuteMsg {
 pub enum QueryMsg {
     #[returns(ChainEndpoint)]
     GetChainInfo(ChainName),
+
+    #[returns(Vec<ChainEndpoint>)]
```

Comment:
Do we want to actually return what should be passed to start_after? Right now the caller needs to extract the last chain from the returned vector and pass it back in. Might be easier to just return a somewhat opaque marker that the caller can just blindly pass into the next call.

Reply:
that's what cosmwasm range query exposed, the api accepts a start key and end key. need to re-implement it otherwise.

Reply:
yeah I understand why they implemented it this way, but I don't like it. I think we'll have to live with it though. And since the query can be called without pagination I don't think we should return a dedicated "bookmark" object. So I suggest leaving the return value as is